cloudnatix/README.md

```diff
@@ -0,0 +1,63 @@
+# Agent Check: CloudNatix
+
+## Overview
+
+This check provides the data from [CloudNatix][1].
+
+CloudNatix connects to multiple VM & K8s Clusters, enabling automated
+capacity, cost & operation optimization using patent-pending Autopilot
+technology. CloudNatix Insights also gives DevOps teams proactive visibility into
+potential capacity and availability issues, turbocharging productivity.
+
+The out-of-the-box integration provides CloudNatix cost and operational
+optimization insights to DataDog and allows showing the data within a
+dashboard.
```

Comment:
Can you include why the cost and operational insights. are helpful (the value)? This should be clear to new users who don't know cloudnatix
```suggestion
The CloudNatix integration provides cost and operational optimization insights to DataDog with an out-of-the-box dashboard. 
```